x-pack/plugins/maps/public/connected_components/mb_map/features_tooltip/header.tsx

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { Component, Fragment, ReactNode } from 'react';
+import { i18n } from '@kbn/i18n';
+import {
+  EuiButtonIcon,
+  EuiHorizontalRule,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiTextColor,
+} from '@elastic/eui';
+
+interface Props {
+  getLayerName: (layerId: string) => Promise<string>;
+  isLocked: boolean;
+  layerId: string;
+  onClose: () => void;
+}
+
+interface State {
+  layerName: string | null;
+}
+
+export class Header extends Component<Props, State> {
+  private _isMounted = false;
+  state: State = { layerName: null };
+
+  componentDidMount() {
+    this._isMounted = true;
+    this._loadLayerName();
+  }
+
+  componentWillUnmount() {
+    this._isMounted = false;
+  }
+
+  async _loadLayerName() {
+    const layerName = await this.props.getLayerName(this.props.layerId);
+    if (this._isMounted) {
+      this.setState({ layerName });
+    }
+  }
+
+  render() {
+    const items: ReactNode[] = [];
+    if (this.state.layerName) {
+      items.push(
+        <EuiFlexItem grow={true} key="layerName">
+          <EuiTextColor>{this.state.layerName}</EuiTextColor>
```

Comment:
discussed offline. Wondering if it would read a little better with a bigger font

![image](https://user-images.githubusercontent.com/1833023/115264319-2fdebe00-a104-11eb-8ada-3524d7810031.png)

Reply:
I wrapped the layer name in `h4` so its now bolded more then column name

<img width="250" alt="Screen Shot 2021-04-19 at 10 59 26 AM" src="https://user-images.githubusercontent.com/373691/115276916-dfb22c80-a100-11eb-8da2-9eb44b27fe40.png">

Reply:
There's so much going on in our tooltips.

Reply:
+1 @kmartastic . fwiw those screenshots are the degenerate case, where there's overlap of multiple layers and multiple features.

I'm not quite sure how we could simplify the common case:

![image](https://user-images.githubusercontent.com/1833023/115306762-456cdb80-a136-11eb-932a-3c0307e4b7ff.png)